src/components/Home/PbA.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  FormattedMessage,
+  FormattedHTMLMessage,
+  injectIntl,
+  intlShape,
+} from 'react-intl'
+import Section from '../General/Section'
+import Juror from './home-assets/juror.svg'
+import background1 from './home-assets/pba-1.png'
+import background2 from './home-assets/pba-2.png'
+import background3 from './home-assets/pba-3.png'
+
+import { breakpoint } from '@aragon/ui'
+const medium = css => breakpoint('medium', css)
+const large = css => breakpoint('large', css)
+
+const PbA = () => (
+  <PbASection>
+    <h3>
+      <FormattedHTMLMessage
+        id="home.pba.title"
+        defaultMessage='Empowering <br /> <span class="blue">innovative communities </span>'
+      />
+    </h3>
+    <Container>
+      <PbACard className="one">
+        <FormattedMessage
+          id="home.pba.card1"
+          defaultMessage="Jesse launched the Streaks DAO to incentivize internal daily usage ofCoinbase apps."
+        />
+      </PbACard>
+      <PbACard className="two">
+        <FormattedMessage
+          id="home.pba.card2"
+          defaultMessage="Jenna launched the Melon Council DAO to expand access to wealth management."
+        />
+      </PbACard>
+      <PbACard className="three">
+        <FormattedMessage
+          id="home.pba.card3"
+          defaultMessage="Esteban launched the Decentraland DAO to enable the first user-governed decentralized virtual world."
+        />
+      </PbACard>
+    </Container>
+    <a className="action-button centered" href="">
```

Comment:
Let's add the link to the page: https://poweredby.aragon.org/

Reply:
Thanks! I didnt know we had a final link for that website :)